lib/core/lib/ops/policyOps.js

```diff
@@ -63,6 +63,15 @@ function deleteUsersAssociations (client, ids, orgId, cb) {
                utils.boomErrorWrapper(cb))
 }
 
+function deleteOrganizationsAssociations (client, ids, orgId, cb) {
```

Comment:
Why the plural (`Organizations`) when there is clearly one `orgId` passed? Same applied to the other associations deletions.